CONTRIBUTING.md

```diff
@@ -0,0 +1,272 @@
+
+[^ Back To The Top](#contributing-to-blockstackorg)
+
+## Pull requests
+
+**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
+
+Good pull requests—patches, improvements, new features—are a fantastic help. They should remain focused in scope and avoid containing unrelated commits.
+
+**Please ask first** before embarking on any significant pull request (e.g. implementing features, refactoring code, porting to a different language), otherwise you risk spending a lot of time working on something that the project's developers might not want to merge into the project.
+
+Ensure that your pull requests are small components. This assists both with your pull request getting accepted and makes it much easier/faster for the repository maintainer to merge in your pull request.
+
+Please adhere to the [Blockstack Brand Guide](https://github.com/blockstack/designs/issues/247), [coding guidelines](#code-guidelines) used throughout the project (indentation, accurate comments, etc.), and any other requirements (such as test coverage).
+
+When contributing to Blockstack's documentation, you should edit the documentation source files in [the `/app/` directory of the `master` branch](https://github.com/blockstack/blockstack.org/tree/master/app).
+
+**Prerequisites:** Make sure you have [git-flow downloaded](https://github.com/petervanderdoes/gitflow-avh/wiki/Installation) on your machine. [Here is a cheatsheet](https://danielkummer.github.io/git-flow-cheatsheet/) to familiarize yourself with the command line tool.
+
+Adhering to the following process is the best way to get your work included in the project:
+
+1. [Fork](https://help.github.com/fork-a-repo/) the project, clone your fork,
+   and configure the remotes:
+
+   ```bash
+   # Clone your fork of the repo into the current directory
+   git clone https://github.com/<your-username>/blockstack.org.git
+   # Navigate to the newly cloned directory
+   cd blockstack.org
+   # Assign the original repo to a remote called "upstream"
+   git remote add upstream https://github.com/blockstack/blockstack.org.git
+   ```
+
+2. If you cloned a while ago, get the latest changes from the development branch upstream.
+
+   ```bash
+   git checkout -b develop
+   git pull upstream develop
+   ```
+
+3. Initiate git flow in the repository and set up a feature branch.
+
+   ```bash
+   git flow init
+   Which branch should be used for bringing forth production releases?
+      - develop
+      - master
+   Branch name for production releases: [master] master
+   
+   Which branch should be used for integration of the "next release"?
+      - develop
+      - master
+   Branch name for "next release" development: [develop] develop
+   
+   How to name your supporting branch prefixes?
```

Comment:
I believe @larrysalibra was the one who suggested the issue number because of githubs ability to reference, I think we could do without it. It would be nice to just doe feature name and github username.